apps/ensapi/src/index.ts

```diff
@@ -51,6 +56,56 @@ app.get("/health", async (c) => {
   return c.json({ ok: true });
 });
 
+// Set default `maxRealtimeDistance` for `GET /amirealtime` endpoint to one minute.
```

Comment:
can we move this into a `src/handlers/amirealtime-api.ts` and mount it like the others? so 

```ts
// in index.ts
app.route('/amirealtime', amirealtimeApi)

// in amirealtime-api.ts
app.get('/', ...)
```

should work